### PR TITLE
Feature/feature change email

### PR DESCRIPTION
**Pull request by danicm47**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/test/java/org/pesmypetcare/mypetcare/features/users/TestChangeMail.java  1
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrChangeMail.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5237263&bid=17166379)

File: app/src/main/res/layout/fragment_settings_menu.xml [hunk 1]
```diff
@@ -3,7 +3,8 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".activities.fragments.NotImplementedFragment">
+    tools:context=".activities.fragments.NotImplementedFragment"
```
Comment (AlbertPG) [inline app/src/main/res/layout/fragment_settings_menu.xml]: Take a look if it's correct

**Review by AlbertPG — APPROVED** [review]
Okey

**Review by enrichp — APPROVED** [review]

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/TrChangeMail.java  1
- app/src/test/java/org/pesmypetcare/mypetcare/features/users/TestChangeMail.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5237263&bid=17166379)

**Pull request merged**